src/test/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/enums/CFTTaskStateTest.java

```diff
@@ -33,4 +37,51 @@ void update_test_whenever_additions_to_assign_enum_are_made() {
         int assigneeEnumLength = CFTTaskState.values().length;
         assertEquals(9, assigneeEnumLength);
     }
+
+    @ParameterizedTest
+    @CsvSource(
+        value = {
+            ",",       // null
+            "''",      // empty
+            "' '",     // blank
+            "123",
+            "null",
+            "some-value"
```

Comment:
can remove the comments :) since they are misleading

Reply:
Maybe it's not understandable without comment :)